sentry-logback/build.gradle.kts

```diff
@@ -0,0 +1,93 @@
+import com.novoda.gradle.release.PublishExtension
+
+plugins {
+    `java-library`
+    kotlin("jvm")
+    jacoco
+    id(Config.QualityPlugins.errorProne)
+    id(Config.Deploy.novodaBintray)
+    id(Config.QualityPlugins.gradleVersions)
+}
+
+configure<JavaPluginConvention> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
+    kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
+}
+
+dependencies {
+    implementation(project(":sentry-core"))
+    // Envelopes require JSON. Until a parse is done without GSON, we'll depend on it explicitly here
+    implementation(Config.Libs.gson)
```

Comment:
ok but sentry-core already has it, do you need to access gson classes explicitly here?

Reply:
You're right, removed.